curvefs/conf/mds.conf

```diff
@@ -64,3 +64,25 @@ mds.heartbeat.intervalMs=10000
 mds.heartbeat.misstimeoutMs=30000
 # heartbeat offline time between metaserver and mds
 mds.heartbeat.offlinetimeoutMs=1800000
+# After mds is started, a certain time delay starts to guide the metaserver to delete data
+mds.heartbeat.clean_follower_afterMs=1200000
+
+#
+# schedule config
+#
+# recoverScheduler switch
+mds.enable.recover.scheduler=false
+# RecoverScheduler round interval, the unit is second
+mds.recover.scheduler.intervalSec=5
+# Concurrency of operator on each metaserver
+mds.schduler.operator.concurrent=1
+# transfer leader timeout, after the timeout, mds removes the operator from the memory
+mds.schduler.transfer.limitSec=60
+# remove a replica timeout, after the timeout, mds removes the operator from the memory
+mds.scheduler.remove.limitSec=300
+# add a replica timeout, after the timeout, mds removes the operator from the memory
+mds.scheduler.add.limitSec=1800
```

Comment:
Add or Remove replica is replaced by Change.

Reply:
New schedule will be added later, this will be retained now.